### PR TITLE
Improve PagerDuty notification doc

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

We've engaged with PagerDuty on a tiny bit of co-marketing, and they had some requests for updates to this document, including a new screenshot and a bit more in the way of instructions.

##### Component Name

area/health

### CODE REVIEW

**Comment by joelhans** [comment]
[PagerDuty Integration](https://trello.com/c/BagpXsLt/415-pagerduty-integration)

**Review by thiagoftsm — DISMISSED** [review]

File: health/notifications/pagerduty/README.md [hunk 1]
```diff
@@ -1,46 +1,69 @@
...
+description: "Send alerts to your PagerDuty dashboard any time an anomaly or performance issue strikes a node in your infrastructure."
+sidebar_label: "PagerDuty"
 custom_edit_url: https://github.com/netdata/netdata/edit/master/health/notifications/pagerduty/README.md
 -->
 
-# PagerDuty
+# Send alert notifications to PagerDuty
 
-[PagerDuty](https://www.pagerduty.com/company/) is the enterprise incident resolution service that integrates with ITOps and DevOps monitoring stacks to improve operational reliability and agility. From enriching and aggregating events to correlating them into incidents, PagerDuty streamlines the incident management process by reducing alert noise and resolution times.
+[PagerDuty](https://www.pagerduty.com/company/) is the enterprise incident resolution service that integrates with ITOps
```
Comment (jlbriston) [inline health/notifications/pagerduty/README.md:10]:
```suggestion
[PagerDuty](https://www.pagerduty.com/company/) is an enterprise incident resolution service that integrates with ITOps
```

File: health/notifications/pagerduty/README.md [hunk 2]
```diff
@@ -1,46 +1,69 @@
...
+notifications](https://user-images.githubusercontent.com/1153921/118317133-872a4100-b4ac-11eb-9cf1-70414aba010f.png)
 
-To have Netdata send notifications to PagerDuty, you'll first need to set up a PagerDuty `Generic API` service and install the PagerDuty agent on the host running Netdata.  See the following guide for details:
+## What you need to get started
 
-<https://www.pagerduty.com/docs/guides/agent-install-guide/>
+- An installation of the open-source [Netdata](/docs/get-netdata.mdx) monitoring agent
+- An installation of the [PagerDuty agent](https://www.pagerduty.com/docs/guides/agent-install-guide/) on the node
+  running Netdata
+- A PagerDuty `Generic API` service using either
```
Comment (jlbriston) [inline health/notifications/pagerduty/README.md:25]: Is there supposed to be more to this sentence? I might be missing something " A PagerDuty `Generic API` service using either"

**Review by jlbriston — APPROVED** [review]

File: health/notifications/pagerduty/README.md (hunk 2 above)
Comment (joelhans) [reply health/notifications/pagerduty/README.md]: Good catches on both of these! Fixed now.

**Review by thiagoftsm — APPROVED** [review]
LGTM!

File: health/notifications/pagerduty/README.md [hunk 3]
```diff
@@ -1,46 +1,67 @@
...
+[PagerDuty](https://www.pagerduty.com/company/) is an enterprise incident resolution service that integrates with ITOps
+and DevOps monitoring stacks to improve operational reliability and agility. From enriching and aggregating events to
+correlating them into incidents, PagerDuty streamlines the incident management process by reducing alert noise and
+resolution times.
 
-Here is an example of a PagerDuty dashboard with Netdata notifications:
+Here's an example of a PagerDuty dashboard with Netdata alert notifications:
 
-![PagerDuty dashboard with Netdata notifications](https://cloud.githubusercontent.com/assets/19278582/21233877/b466a08a-c2a5-11e6-8d66-ee6eed43818f.png)
+![PagerDuty dashboard with Netdata alert notifications](https://user-images.githubusercontent.com/1153921/118317133-872a4100-b4ac-11eb-9cf1-70414aba010f.png)
```
Comment (ilyam8) [inline health/notifications/pagerduty/README.md:17]: [The screenshot](https://user-images.githubusercontent.com/1153921/118317133-872a4100-b4ac-11eb-9cf1-70414aba010f.png) is [very blurry](https://i.kym-cdn.com/photos/images/newsfeed/002/117/476/261.jpg).

Comment (ilyam8) [reply health/notifications/pagerduty/README.md:17]: What do we do? @kickoke 

- close this PR.
- remove the screenshot, merge the PR.
- keep the screenshot, merge the PR.

Comment (kickoke) [reply health/notifications/pagerduty/README.md:17]: I would remove the screenshot and then merge the PR. 
We struggle with maintaining our own screenshots, so I don't want to add maintaining 3rd party screenshots to the list.
Thank you @ilyam8

**Review by ilyam8 — APPROVED** [review]

**Review by kickoke — APPROVED** [review]

**Pull request merged**